docs/connections.jade

```diff
@@ -9,7 +9,9 @@ block content
     mongoose.connect('mongodb://localhost/myapp');
 
   :markdown
-    This is the minimum needed to connect the `myapp` database running locally on the default port (27017). We may also specify several more parameters in the `uri` depending on your environment:
+    This is the minimum needed to connect the `myapp` database running locally on the default port (27017). If the connection fails or if attempting to connect via a Mongo GUI only shows a blank screen despite successful connection, then try using 127.0.0.1 instead of localhost. Sometimes issues may arise when the local hostname has been changed.
```

Comment:
I'd argue that the GUI stuff shouldn't be in here. IMO those docs really belong in the GUI's docs, especially since you can't be entirely sure that's how all mongodb GUI's work on all OS.